app-misc/splitvt/splitvt-1.6.6-r4.ebuild

```
@@ -0,0 +1,53 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MY_P="${P/-/_}"
+DEB_PL="7"
+
+DESCRIPTION="Splitting terminals into two shells"
+HOMEPAGE="https://slouken.libsdl.org/projects/splitvt/"
+SRC_URI="
+	https://slouken.libsdl.org/projects/splitvt/${P}.tar.gz
+	mirror://debian/pool/main/s/splitvt/${MY_P}-${DEB_PL}.diff.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ia64 ~ppc ~sparc ~x86"
+IUSE=""
+
+DEPEND="sys-libs/ncurses:0="
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${WORKDIR}"/${MY_P}-${DEB_PL}.diff
+	"${FILESDIR}"/${PV}-ldflags.patch
+	"${FILESDIR}"/${PV}-fix-build-for-clang16.patch
+)
+
+DOCS=( ANNOUNCE BLURB CHANGES NOTES README TODO )
+
+src_prepare() {
+	default
+	sed -i \
+		-e "s|/usr/local/bin|${EPREFIX}/usr/bin|g" \
+		-e "s|/usr/local/man/|${EPREFIX}/usr/share/man/|g" config.c || die
+}
+
+src_configure() {
+	# upstream has their own weirdo configure script...
```

I usually like to rephrase stuff like this so we look a bit more professional:
```suggestion
	# upstream has their own homebrew configure script
```